### PR TITLE
Documentation with VitePress

### LINKED ISSUE #33: Deployment of docs and demo apps
- [ ] Automate deployment of Documentation and demo apps for Angular, Vue and React
- [ ] Setup Github Action for documentation publishing
- [ ] Setup Github pipeline for building and running documentation build on feature branch

### PR DESCRIPTION
**Pull request by ibirrer**
Closes #33

### CODE REVIEW

File: docs/guide/angular.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   ```scss
+   @import '@six-group/ui-library/dist/ui-library/ui-library.css';
+   ```
+
+## Using the Components
+
+The components can be utilized just like any other Angular component. However, there's one caveat:
+while you'll receive code completion for attributes/properties of the component, code completion for
+events provided by the components is currently unavailable. Please check the component documentation
+(TODO) for available events.
```
Comment (stefanrufer) [inline docs/guide/angular.md:37]: Fix TODO

**Review by stefanrufer — CHANGES_REQUESTED** [review]
two/three comments in the guide part.

File: docs/guide/angular.md (hunk 1 above)
Comment (ibirrer) [reply docs/guide/angular.md]: Done

File: docs/guide/upgrade-v4.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+In version 4, we have made a strategic decision to remove the `six-table` component from the
+library. This decision is in line with our commitment to refining core functionalities and
+simplifying the library in general. While offering a comprehensive table component is valuable, the
+complexity involved is better suited for dedicated solutions.
+
+As a result, we recommend exploring existing table components within your framework's ecosystem to
+find the best-fit solution for your needs. For instance, in certain Angular projects, we've found
+success with [AG Grid](https://www.ag-grid.com/), a powerful and flexible table component that
+offers extensive capabilities.
```
[the remark this answers is not in the document]
Comment (colinscz) [reply docs/guide/upgrade-v4.md:93]: I think there is no harm in mentioning the name of that library that exist internally. One thing that would be crucial to have is an easy way of finding that documentation and how to integrate this internal library. 

Maybe we can reference it in the internal teams channel or on our internal confluence. That way we can just say something like "find more information about this on our internal confluence space for the six webcomponents". 

@Riatomerehentroweard any suggestions from your side?

**Review by stefanrufer — APPROVED** [review]

**Pull request merged**